dali/python/nvidia/dali/fn.py

```diff
@@ -77,7 +77,7 @@ def op_wrapper(*inputs, **arguments):
 
     op_wrapper.__name__ = wrapper_name
     op_wrapper.__qualname__ = wrapper_name
-    op_wrapper.__doc__ = "see :class:`{0}.{1}`".format(op_class.__module__, op_class.__name__)
+    op_wrapper.__doc__ = op_class.__doc__
```

Comment:
Use documentation from `__call__` instead - class documentation is basically `__init__` and it doesn't have any notion of positional inputs.